### PR TITLE
NIFI-5137 - Fixed the path to controller service grid item's state

### PR DESCRIPTION
**Pull request by annanys23**
# Summary

[NIFI-5137] (https://issues.apache.org/jira/browse/NIFI-5137)
In nf-controller-services.js fixed the path to controllerServiceEntity.component.state to check for when the controller service is disabled and state can be cleared.  This now enables the 'Clear state' link when the service is disabled.

# Tracking

Please complete the following tracking steps prior to pull request creation.

### Issue Tracking

- [x] [Apache NiFi Jira](https://issues.apache.org/jira/browse/NIFI) issue created

### Pull Request Tracking

- [x] Pull Request title starts with Apache NiFi Jira issue number, such as `NIFI-00000`
- [x] Pull Request commit message starts with Apache NiFi Jira issue number, as such `NIFI-00000`

### Pull Request Formatting

- [x] Pull Request based on current revision of the `main` branch
- [x] Pull Request refers to a feature branch with one commit containing changes

# Verification

Please indicate the verification steps performed prior to pull request creation.

### Build

- [x] Build completed using `mvn clean install -P contrib-check`
  - [x] JDK 21

### Licensing

- [ ] New dependencies are compatible with the [Apache License 2.0](https://apache.org/licenses/LICENSE-2.0) according to the [License Policy](https://www.apache.org/legal/resolved.html)
- [ ] New dependencies are documented in applicable `LICENSE` and `NOTICE` files

### Documentation

- [x] Documentation formatting appears as expected in rendered files

### CODE REVIEW

**Comment by NissimShiman** [comment]
Reviewing...

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-controller-services.js [hunk 1]
```diff
@@ -1090,7 +1090,7 @@
                 } else if (target.hasClass('delete-controller-service')) {
                     nfControllerService.promptToDeleteController(serviceTable, controllerServiceEntity);
                 } else if (target.hasClass('view-state-controller-service')) {
-                    nfComponentState.showState(controllerServiceEntity, controllerServiceEntity.state === 'DISABLED');
+                    nfComponentState.showState(controllerServiceEntity, controllerServiceEntity.component.state === 'DISABLED');
```
Comment (NissimShiman) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-controller-services.js:1093]: @annanys23 I tried it and it works.  Very nice!

I was thinking maybe there should be a check for activeThreadCount as well.  We do this for [reportingTasks](https://github.com/apache/nifi/blob/rel/nifi-1.23.2/nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-settings.js#L1793-L1794)

So we would have something like:
```
var canClear = controllerServiceEntity.status.runStatus === 'DISABLED' && (nfCommon.isUndefinedOrNull(controllerServiceEntity.status.activeThreadCount) || controllerServiceEntity.status.activeThreadCount === 0);
nfComponentState.showState(controllerServiceEntity, canClear);
```

Comment (annanys23) [reply nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-controller-services.js:1093]: Thanks for testing!  Controller services, unlike reporting tasks and processors do not have activeThreads.  When a controller service is disabled, all connected components are stopped.  So, activeThreadCount for those should be 0.

**Review by exceptionfactory — APPROVED** [review]
Thanks for finding and fixing this problem @annanys23, and thanks for testing the solution @NissimShiman! The correction looks good, and aligns as expected with the location of the `state` property. +1 merging

**Pull request closed without merging**